Debian/8/templates/template_partitions_rights

```diff
@@ -0,0 +1,26 @@
+<def-group>
+  <definition class="compliance" id="mount_optionPOINTID_FLAGID" version="1">
```

Comment:
I concatenated because the first "/" of the mountpoint is substituted by a '_'. Adding an underscore here will generate an id with 2 concatenated underscores.
If you prefer I can update the POINTID value to delete the starting '_'.